client/src/components/Home.js

```diff
@@ -40,7 +40,7 @@ const Home = () => {
 export default Home;
 
 const ChannelContainer = styled.div`
-  padding: 10px;
+  padding-top: 30px;
```

Comment:
doesn't this remove the 10px on other sides?

Reply:
I didn't notice much difference. Does it look weird on your end with "padding: 10px" instead of "padding-top: 30px"?